fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanUtils.ts

```diff
@@ -5,6 +5,7 @@
 import { PathVector } from '@flybywiresim/fbw-sdk';
 import { ReadonlyFlightPlan } from './plans/ReadonlyFlightPlan';
 import { ReadonlyFlightPlanElement } from './legs/ReadonlyFlightPlanLeg';
+import { LnavConfig } from '../guidance/LnavConfig';
 
 export class FlightPlanUtils {
   public static getAllPathVectorsInFlightPlan(
```

Comment:
This should be renamed to indicate that it's only returning the EFIS path vectors. But then, does it even belong in "FlightPlanUtils"? It's not really to do with flight planning.